MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -750,10 +750,13 @@ private TranscodingProfile GetVideoTranscodeProfile(MediaSourceInfo item, VideoO
                         var container = transcodingProfile.Container;
                         var appliedVideoConditions = options.Profile.CodecProfiles
                             .Where(i => i.Type == CodecType.Video &&
-                                i.ContainsAnyCodec(videoCodec, container))
+                                i.ContainsAnyCodec(videoCodec, container) &&
```

Comment:
Style-wise, we put the `&&` before the lines, not after

Reply:
Then this is what it would look like (since I don't want to touch more lines than enough):
```c#
.Where(i => i.Type == CodecType.Video &&
    i.ContainsAnyCodec(videoCodec, container)
    && i.ApplyConditions.All(...))
```

And below:
```c#
.Where(codecProfile => codecProfile.Type == CodecType.Video && codecProfile.ContainsAnyCodec(videoStream?.Codec, container)
    && !checkVideoConditions(codecProfile.ApplyConditions).Any())
```